spec/controllers/reports_controller_spec.rb

```diff
@@ -358,6 +358,14 @@
         expect(response).to render_template("new")
       end
 
+      it "copies the comment id into a hidden form field" do
+        get :new, :request_id => info_request.url_title,
+                  :comment_id => comment.id
+        expect(response.body).
```

Comment:
next commit fixes this spec by adding the missing  `render_views` so going to discard the commit message about the failure